presto-spi/src/main/java/com/facebook/presto/spi/type/CharType.java

```diff
@@ -30,6 +30,7 @@
         extends AbstractVariableWidthType
 {
     public static final int MAX_LENGTH = 65_536;
+    public static final CharType CHAR = new CharType(MAX_LENGTH);
```

Comment:
Not sure. Do we need it just for test?
Seems current implementation Char type without length specified has default length of 1.